src/compiler/abstract_compiler.nit

```diff
@@ -1594,39 +1594,39 @@ abstract class AbstractCompilerVisitor
 	fun int8_instance(value: Int8): RuntimeVariable
 	do
 		var t = mmodule.int8_type
-		var res = new RuntimeVariable("((int8_t){value.to_s})", t, t)
+		var res = new RuntimeVariable("INT8_C({value.to_s})", t, t)
```

Comment:
The old code used an `int` literal and cast it to a `int8_t`, the change here uses directly a literal `int8_t`. This was probably optimized by the C compiler already but now we skip a step and the style is constant with the `int_fast64_t` literal.